CHANGELOG.md

```diff
@@ -7,13 +7,20 @@ Please enumerate all user-facing changes using format `<githib issue/pr number>:
 
 ## 0.7.0
 
+* [#143](https://github.com/kroxylicious/kroxylicious-junit5-extension/issues/143): Default native kafka image tag from the version of Kafka on the classpath.
 * [#197](https://github.com/kroxylicious/kroxylicious-junit5-extension/issues/197): Consumer of the extension should be able to choose the version of the kafka dependency.
 
-### Changes, deprecations and removals
 
-* It is now necessary to declare org.apache.kafka:kafka_2.13 as a test dependency.  If you wish to use kafka in-vm with zookeeper, org.apache.zookeeper:zookeeper
-  must be a test dependency too.
+### Changes, deprecations and removals
 
+* It is now necessary to declare `org.apache.kafka:kafka_2.13` as a test dependency.  If you wish to use kafka in-vm with
```

Comment:
It could just be my poor comprehension skills but I think this should really be two paragraphs.